README.md

```diff
@@ -10,6 +12,18 @@ The Azure cloud provider code locates at [Kubernetes repository directory](https
 
 There is an ongoing work for refactoring cloud providers out of the upstream repository. For more details, please check [this issue](https://github.com/kubernetes/features/issues/88).
 
+## Current status
+
+cloud-provider-azure is still under **alpha** stage and its releases are maintained on Microsoft Container Registry (MCR).
+
+The latest version of azure-cloud-controller-manager could be found at `mcr.microsoft.com/k8s/core/azure-cloud-controller-manager:v0.1.0`.
+
+Version matrix:
+
+|Kubernetes version|cloud-provider version|cloud-provider branch|
+|------------------|----------------------|---------------------|
+| v1.14.x          | v0.1.0               | master              |
```

Comment:
For `cloud-provider branch`, should we use `master` or would a tag be better?

Reply:
We only have master now, after GA, we may have release branches, e.g. release-1.0